falk/vm/docker.py

```diff
@@ -3,3 +3,59 @@
 
 https://www.docker.com/
 """
+
+import logging
+import os
+from pathlib import Path
+
+from . import Container, ContainerConfig
+import docker
+
+
+class Docker(Container):
+    """
+    Represents a Docker container.
+    """
+
+    def __init__(self, cfg):
+        super().__init__(cfg)
+        self.manage = False
+        self.docker_container_id = None
+
+    @classmethod
+    def config(cls, machine_id, cfgdata, cfgpath):
+        cfg = ContainerConfig(machine_id, cfgdata, cfgpath)
+
+        cfg.dockerfile = Path(cfgdata["dockerfile"])
+        cfg.image_name = Path(cfgdata["image_name"])
```

Comment:
If the path is relative, you should make it relative to `cfgpath` so it remains intuitive to use.